app/enterprise/1.5.x/kong-for-kubernetes/install.md

```diff
@@ -1,33 +1,32 @@
 ---
-title: Installing Kong for Kubernetes Enterprise
+title: Installing Kong Enterprise with Kubernetes
 ---
 
 ## Introduction
-This installation topic guides you through installing and deploying Kong for Kubernetes Enterprise (K4K8S-Enterprise), then directs you to the documentation for configuring and using the product.
+Kong for Kubernetes provides two options for its Kong Enterprise container image:
 
- You can install Kong for Kubernetes Enterprise using YAML with kubectl, or with OpenShift oc. Other deployment options, such as using Helm Chart and Kustomize, will be available at a later time.
+* Kong for Kubernetes Enterprise (`kong-enterprise-k8s`): provides most Kong Enterprise plugins and runs without a database, but does not include other Kong Enterprise features (Kong Manager, Dev Portal, Vitals, etc).
```

Comment:
`kong-enterprise-k8s` is a behind the scenes name. Please remove it from this stanza.

Reply:
It does get exposed in the bintray link later in the topic though, so users are aware of it. The new deployment topic also refers to this name (pr #2015 )

If I split this topic out anyway, there will be no need to refer to this name in the intro, but something to be aware of for the other topic.

Reply:
I think the split is the best way to go. There are two concerns we want to address:
* Users have difficulty understanding the differences between the products and knowing which image to specify. My doc is focused on comparing them and listing the images prominently.
* Users want a step-by-step guide for installing their chosen product, which this provides. Splitting this into two documents duplicates a lot (and I don't think Jekyll provides a great way to write and reuse the shared sections), but is probably best to reduce decision points.

Reply:
Further complicating the matter is the controller-managed versus not split: after reading I realized that the new documentation actually refers to the latter. I'd lean towards ignoring that for the most part and focusing on the DB-less (Kong for Kubernetes Enterprise) and DB-backed (Kong for Kubernetes with Kong Enterprise), with DB-backed, no controller clusters (Kong Enterprise on Kubernetes) in either a minor note or separate article.

@hbagdi aside from the article name, the only item of concern at installation should be whether or not the controller is enabled, correct? As far as I know there aren't any other settings that differ; all the Manager/Portal/RBAC/etc. configuration should be the same, though we should instruct users to create and configure an RBAC user for the controller.

Reply:
Addressed this by referring to db-backed vs db-less in the topic introductions and linking off to the the deployment topic for further info.